setup.py

```diff
@@ -43,6 +42,16 @@
         name=name,
         version=package_info["VERSION"],
         description="Insights Application Programming Interface",
+        author_email="insights@redhat.com",
+        license="Apache 2",
+        url="https://access.redhat.com/insights",
+        classifiers=[
+            "Development Status :: 5 - Production/Stable",
+            "License :: OSI Approved :: Apache Software License",
```

Comment:
@kylape this may not be necessary but should the license be specified as `Apache License, Version 2.0`, here and above for `license`?